openlibrary/i18n/messages_cs.po

```diff
@@ -0,0 +1,2179 @@
+
+#: forms.py:10 templates/login.html:33
+msgid "Password"
+msgstr "Heslo"
+
+#: forms.py:15
+msgid "No user registered with this email address"
+msgstr "Nebyl nalezen uživatel s tímto emailem"
+
+#: forms.py:16
+msgid "Email already used"
+msgstr "Email už je používán"
+
+#: forms.py:17
+msgid "Username already used"
+msgstr "Uživatelské jméno už je použito"
+
+#: forms.py:19
+msgid "Must be between 3 and 20 letters and numbers"
+msgstr "Musí mýt 3 až 20 písmen a čísel"
+
+#: forms.py:20
+msgid "Must be between 3 and 20 characters"
+msgstr "Musí mýt 3 až 20 znaků"
+
+#: forms.py:21
+msgid "Must be a valid email address"
+msgstr "Musí být platná emailová adresa"
+
+#: forms.py:24 forms.py:57
+msgid "Your Email Address"
+msgstr "Vaše emailová adresa"
+
+#: forms.py:25 templates/account/create.html:40
+msgid "Choose a Username"
+msgstr "Vyberte si uživatelské jméno"
+
+#: forms.py:25
+msgid "Only letters and numbers, please, and at least 3 characters."
+msgstr "Prosím, pouze písmena a čísla, a nejméně 3 znaky."
+
+#: forms.py:27 forms.py:61
+msgid "Choose a Password"
+msgstr "Vyberte heslo"
+
+#: forms.py:45
+msgid "Invalid password"
+msgstr "neplatné heslo"
+
+#: forms.py:48
+msgid "Current Password"
+msgstr "současné heslo"
+
+#: forms.py:49
+msgid "Choose a New Password"
+msgstr "Vyberte si nové heslo"
+
+#: forms.py:53 templates/account/email.html:20
+msgid "Your New Email Address"
+msgstr Vaše nová meailová adresa""
+
+#: utils.py:232
+#, python-format
+msgid "%d "
+msgstr ""
+
+#: macros/EditButtons.html:5
+msgid "Edit Summary (briefly describe the changes you have made):"
+msgstr "Shrnutí editace (stručně popište změny které jste provedly):"
+
+#: macros/EditButtons.html:14 macros/EditButtonsMacros.html:25
+#: templates/account/notifications.html:109 templates/lib/EditButtons.html:24
+#: templates/lib/EditButtonsMacros.html:25
+msgid "Save"
+msgstr "Uložit"
+
+#: macros/EditButtons.html:16 templates/editpage.html:11
+#: templates/type/permission/edit.html:15
+msgid "Preview"
+msgstr "Náhled"
+
+#: macros/EditButtons.html:18 macros/EditButtonsMacros.html:27
+#: templates/login.html:47 templates/work_search.html:246
+#: templates/account/create.html:73 templates/account/email.html:33
+#: templates/account/notifications.html:111 templates/account/password.html:42
+#: templates/account/password/forgot.html:23
+#: templates/account/password/reset.html:20 templates/books/add.html:68
+#: templates/books/edit/addcover.html:32 templates/lib/EditButtons.html:30
+#: templates/lib/EditButtonsMacros.html:27
+msgid "Cancel"
+msgstr "Zrušit"
+
+#: macros/EditButtons.html:20
+msgid "Delete"
+msgstr "Smazat"
+
+#: macros/EditButtons.html:23
+msgid ""
+"Except where otherwise noted, content on this site is licensed under a <a"
+" href=\"http://creativecommons.org/choose/zero\">Creative Commons CC0 "
+"License"
+msgstr ""
+"Kde není uvedeno jinak,je obsah těchto stránek licencován <a"
+" href=\"http://creativecommons.org/choose/zero\">Creative Commons CC0"
```

Comment:
Shouldn't this be https://creativecommons.org/choose/zero/?lang=cs instead?

Reply:
Good question to raise @LeadSongDog , I'm not sure it is necessary though if the user has browser language settings configured. I tested OL using Czech browser settings, and following the standard CC0 url was presented with some Czech (mainly navigation, the CC main content was still in English, but that's their issue).

I am loving the browser language setting option that @tfmorris brought to my attention. It means we do not have to code any custom settings pages or manage extra data. In addition to supporting my laziness, it does seem to be the way the web should work.

We need to decide (at some point) if OL needs prominent per session language links (which would advertise the feature), and account language preferences, but for now I am content with browser settings being the way to use and test i18n. I don't regularly use this sort of feature though, so data + opinions from users would be great to drive these decisions!

Reply:
Browsers typically inherit from the operating system, extending the laziness. :-) My browser allows for an ordered list of fallback languages, which I think allows for a lot of flexibility/power without resorting to special case OL-specific code.